driver/main_args.ml

```diff
@@ -1816,6 +1823,12 @@ module Default = struct
         global_string_constants := (name, value) :: !global_string_constants
       | exception Not_found ->
         Compenv.fatal "Expect <name>=<value> for -set-global-string"
+    let _set_runtime_default s =
+      match Misc.cut_at s '=' with
+      | (name, value) ->
+         runtime_parameters := (name, value) :: !runtime_parameters
```

Comment:
More validation needed here - we shouldn't leave validating both the keys and the values to startup_aux.c (i.e. there needs to be something which knows what _all_ the runtime parameter are, and what their valid values should be). It'd be neater as well if this was eliminating duplicates - i.e. `-set-runtime-parameter b=1 -set-runtime-parameter b=0` should result in only `b=0` going into the file.